client/chatbot.js

```diff
@@ -19,7 +19,7 @@ const Chatbot = () => {
       <ThemeProvider theme={theme}>
         <ChatBot
           floating="true"
-          botAvatar="connor.jpeg"
+          botAvatar="connor.jpg"
```

Comment:
There is a really small bug with the chatbot.  If you refresh the page when in the cart or checkout page, it can't find connor's img.  I tried fixing it, but its a bundle issue

Reply:
I know. I'll get to this later